typescript/packages/core/src/plugins/send-nft.ts

```diff
@@ -0,0 +1,64 @@
+import { Connection, PublicKey } from "@solana/web3.js";
+import { parseUnits } from "viem";
+import type { SolanaWalletClient } from "../wallets";
+import type { Plugin } from "./plugins";
+import { z } from "zod";
+import { SystemProgram } from "@solana/web3.js";
+import {
+	mplBubblegum,
+	getAssetWithProof,
+	transfer,
+} from "@metaplex-foundation/mpl-bubblegum";
+import { createUmi } from "@metaplex-foundation/umi-bundle-defaults";
+import {
+	fromWeb3JsPublicKey,
+	toWeb3JsInstruction,
+} from "@metaplex-foundation/umi-web3js-adapters";
+
+export function transferNFT(): Plugin<SolanaWalletClient> {
```

Comment:
Two things here:
1. I would put this on its own package, maybe `packages/plugins/solana-nfts`
2. If there are other nft functions that we would add (e.g for evm we could have approveAll, balanceOf, etc.), I would name the plugin `nfts`,  not `transferNFTs` so the import would be somthing like:

```typescript
const tools = getOnChainTools({
  wallet: solana(wallet),
  plugins: [
     nfts(), // Adds all nft related functions, transfer, approve, etc.
  ]
});

Reply:
That sounds fine to me, but one of the reasons I made this it's own plugin is because of what you mentioned about LLMs wanting to use all provided functions even if they aren't necessary. In your experience is this effect minor enough that it's ok to do so for the sake of convenience?

Reply:
Right, I'm guessing that if you are doing things with nfts you probably won't just transfer them, but maybe get metadata and so

Reply:
Ok, changed